content/docs/tasks/security/vault-ca/index.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: Istio Vault CA Integration
-description: Tutorial on how to plug in a Vault CA for issuing certificates in Istio.
+description: Tutorial on Istio Vault CA integration and a demo of Istio mutual TLS using certificates from a Vault CA.
```

Comment:
Let's not use the word "Tutorial" here, that's got a different connotation.
Similar, let's not use "demo". 

I suggest simply:

Show how to integrate with Vault CA for mutual TLS.

Reply:
Done.